lib/elixir/lib/kernel.ex

```diff
@@ -3271,7 +3271,12 @@ defmodule Kernel do
       if Map.has_key?(fields, :message) do
         @spec message(Exception.t) :: String.t
         def message(exception) do
-          exception.message
+          cond do
+            is_nil(exception.message) ->
```

Comment:
I mean improve this message
```
** (ArgumentError) argument error
    :erlang.byte_size(nil)
```
to something saying that provided message is `nil`.
It feels a bit magical to have "" default here, even if exception would be defined as `defexception message: nil`.
I think user could explicitly define default: `message: ""` if one wants this default, otherwise it should fail with clear error.